index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <link rel="stylesheet" href="style.css">
+    <title>Kossi Fioklou Portfolio</title>
+</head>
+<body>
+    <!--This is the section for the header-->
+    <header class="header">
+      <nav id="nav-bar">
+        <div id="logo">
+            <a href="#">
+                <img src="assets/My_Logo.png" alt="logo">
+            </a> 
+        </div>
+          <div class="nav-buttons">
+            <a href="#works" class="nav-button">Portfolio</a>
+            <a href="#about" class="nav-button">About</a>
+            <a href="#contact" class="nav-button">Contact</a>
+          </div>
+           <div class="fontawesome">
+            <i class="fa fa-bars"></i>
+           </div>
+      </nav>
+    </header>
+    <!--Main section-->
+
+   <main>
+    <section id="main-page">
+      <div class="header-content">
+      <h1 class="header-text">I'm Kossi Fioklou <br> Glad to see you! </h1>
+      <p class="tonic-para">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact
+        me.
+      </p>
+      <div class="social-media">
+        <h2 class="social-media-text">LET'S CONNECT</h2>
+        <div class="social-media-icons"></div>
```

Comment:
- [ ] Optional
   * Please delete this div, it seems like it contains nothing div